ui/js/models/resultsets_store.js

```diff
@@ -288,7 +287,7 @@ treeherder.factory('ThResultSetStore', [
                 }
                 return false;
             };
-            _.find(getJobMap(), addIfShown);
```

Comment:
Ends up that ``find`` is not a function of the return from ``getJobMap()``.  That's why the selenium test failed.  Would you debug that and submit an update?

Reply:
Hi, I believe this is working now. I've implemented it using a for in loop instead of using .find.